docs/guides/admin/mkdocs.yml

```diff
@@ -174,6 +173,7 @@ nav:
    - Wowza Streaming Distribution Service: 'configuration/streaming-wowza.md'
    - YouTube Publication: 'configuration/youtubepublication.md'
 - Version Support: 'version-support.md'
+- Domain change/HTTPS migration: 'migration-domain-https.md'
```

Comment:
This might be a bit too high level.  What about under Configuration, at the same level as `HTTPS` and `Database`?  Then it's findable (ie, not hidden in HTTPS itself), but not super-duper header level important?